cyclus/lib.pyx

```diff
@@ -1808,44 +1826,63 @@ cpdef dict normalize_request_portfolio(object inp):
     if not isinstance(inp, Mapping):
         inp = dict(inp)
     if 'commodities' in inp:
-        commods = inp['commodities']
+        commods = []
+        prefs = []
```

Comment:
I don't think you need this variable

Reply:
I removed the `prefs` list, because that is unused. The `commods` list is used, so that remains.